### DISCUSSION TITLE
Placeholder component with relationship

### DISCUSSION THREAD

**Discussion by ShaungBhone** [filamentphp/filament#14032] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

I'm trying to show all the value when user creating invoice in the wizard. User name is not showing like invoice belongsTo user. How can I show invoice belong to user?
```
Forms\Components\Wizard\Step::make('Reviews')
  ->schema([
    Forms\Components\Placeholder::make('number')
      ->content(fn($get) => $get('number')),
          Forms\Components\Placeholder::make('user') <-- not working give me null
            ->content(fn($get) => $get('name')),
               Forms\Components\Placeholder::make('note')
                    ->content(
                       fn($get)
                        => new HtmlString($get('note'))
                   ),
]),
```

**Comment by ShaungBhone** [comment]
No one helps me. Nice

**Comment by e2d2-dev** [comment]
Hi,

do You have a Select or something, where the state is stored? When this snippet is all of Your code, then it's missing a lot. I prefer to use a Select an do a Model::query()->pluck('name', 'id') or map() if the name should contain more than one attribute like CustomerID | Name.

Hope that helps, if not, more information needed.


ciao ed

**Reply by ShaungBhone** [reply]
It is wizard to show data after user create the value.

**Reply by e2d2-dev** [reply]
You still need to fetch the data, where is the user_id stored? To fetch data you'd still need to do e.g.

`User::find($get('user_id'))->name`

to retrieve the name.

**Reply by ShaungBhone** [reply]
It's not working. I leave it now.

**Comment by rvzug** [comment]
```php
Forms\Components\Placeholder::make('user')
    ->content(fn(Model $record) => $record->user->name)
```

Should work, if i'm rught. Assuming that the form-record has a ->user() relation defined.
